analytics/graph_record.go

```diff
@@ -30,29 +31,29 @@ type GraphRecord struct {
 	HasErrors     bool         `gorm:"has_errors"`
 }
 
-func (a *AnalyticsRecord) ToGraphRecord() (GraphRecord, error) {
-	record := GraphRecord{
-		AnalyticsRecord: *a,
-		RootFields:      make([]string, 0),
-		Types:           make(map[string][]string),
-	}
-	if a.ResponseCode >= 400 {
-		record.HasErrors = true
+func parseRequest(encodedRequest, encodedSchema string, record *GraphRecord) {
```

Comment:
@kofoworola please let's refactor this to be part of `GraphRecord` and add some comments 🙏

Reply:
Thanks for pointing this, addressed